src/PreCICE.jl

```diff
@@ -573,12 +564,13 @@ Previous calls:
 # Examples
 ```julia
 vertices = [1,1,1,2,2,2,3,3,3]
-vertex_ids = setMeshVertices(mesh_id, 3, vertices)
+vertex_ids = setMeshVertices(mesh_id, vertices)
 ```
 """
-function setMeshVertices(meshID::Integer, size::Integer, positions::AbstractArray{Float64})
-    vertexIDs = Array{Int32, 1}(undef, size)
-    ccall((:precicec_setMeshVertices, libprecicePath), Cvoid, (Cint, Cint, Ref{Cdouble}, Ref{Cint}), meshID, size, positions, vertexIDs)
+function setMeshVertices(meshID::Integer, positions::AbstractArray{Float64})
+    _size = Cint(length(positions)/PreCICE.getDimensions())
```

Comment:
Ideally if the correct positions and dimensions are entered then this variable will always be an int. Is the enforced data type a requirement?

Reply:
The [division operator](https://docs.julialang.org/en/v1/base/math/#Base.:/) returns a float by default. There are other types of division, like [rational division `//`](https://docs.julialang.org/en/v1/base/math/#Base.://) and also [integer division](https://docs.julialang.org/en/v1/base/math/#Base.div) which makes either use of the divide symbol `÷` which I don't have on my keyboard or use `div(a,b)`. This would probably be possible, because [Julia casts autmatically](https://docs.julialang.org/en/v1/base/c/#Base.@ccall) from Int32 to Int64. The division returns Int64 while [Cint is equivalent to Int32](https://docs.julialang.org/en/v1/base/c/#Base.Cint). Automated tests would be helpful here.